wrangler-core/src/test/java/io/cdap/directives/aggregates/SetTransientVariableTest.java

```diff
@@ -50,6 +50,7 @@ public void testSettingOfVariable() throws Exception {
     rows.add(new Row("fwd", 1).add("A", 2));
 
     final Map<String, Object> s = new HashMap<>();
+    s.put("test", 1);
```

Comment:
is this intended? don't see any change to the rest of the test

Reply:
unintended.